bundle.cmake

```diff
@@ -116,12 +117,18 @@ function(QURT_BUNDLE)
 		foreach(inc ${FASTRPC_ARM_LINUX_INCLUDES})
 			list(APPEND ${QURT_BUNDLE_APP_NAME}_INCLUDE_DIRS -I${inc})
 		endforeach()
+		
+    	# prepend -I in front of QAIC include dirs
+    	set(QAIC_INCLUDE_DIRS)	
+    	foreach(inc ${QURT_BUNDLE_QAIC_INCS})
+    		list(APPEND QAIC_INCLUDE_DIRS -I${CMAKE_CURRENT_SOURCE_DIR}/${inc})
```

Comment:
This won't work if someone passes an absolute path. Need to check if path starts with "/".